CONTRIBUTING.md

```diff
@@ -0,0 +1,362 @@
+# Contributing Guidelines
+
+
+[`Ergo Main Repo`](https://github.com/ergoplatform/ergo) 
+
+Want to hack on Ergo? Awesome! Here are instructions to get you started. They are not perfect yet. 
+Please let us know what feels wrong or incomplete.
+
+Ergo is an Open Source project and we welcome contributions of all sorts. 
+There are many ways to help, from reporting issues, contributing code, 
+and helping us improve our community.
+
+## Topics
+
+- [Security Issues](#security-issues)
+- [Community Guidelines](#community-guidelines)
+- [Dive Right In](#dive-right-in)
+- [Reporting Issues](#reporting-issues)
+- [Protocol Design](#protocol-design)
+- [Implementation Design](#implementation-design)
+- [Community Improvement](#community-improvement)
+- [Translations](#translations)
+- [Helping in other ways](#helping-in-other-ways)
+- [Code Contribution Guidelines](#code-contribution-guidelines)
+  - [Discuss big changes as Issues first](#discuss-big-changes-as-issues-first)
+  - [Pull Requests always welcome](#pull-requests-always-welcome)
+  - [Conventions](#conventions)
+    - [Git](#git)
+    - [Commit messages](#commit-messages)
+      - [Subject line should not be more than 80 characters long](#subject-line-should-not-be-more-than-80-characters-long)
+      - [A "License" and a "Signed-off-by" trailers are required](#a-license-and-a-signed-off-by-trailers-are-required)
+      - [Commit message examples](#commit-message-examples)
+    - [Code](#code)
+    - [Tests](#tests)
+    - [Documentation](#documentation)
+    - [Pull Requests](#pull-requests)
+    - [Code Review](#code-review)
+    - [Rebasing](#rebasing)
+    - [Merge Approval](#merge-approval)
+- [Credits](#credits)
+
+
+## Security Issues
+
+The Ergo protocol and its implementations are still in heavy development. 
```

Comment:
The core protocol is done, for networking and application protocols (client interfaces and so on) a lot of work is needed for sure.